Userland/Applications/PixelPaint/Layer.cpp

```diff
@@ -135,10 +135,19 @@ RefPtr<Gfx::Bitmap> Layer::try_copy_bitmap(Selection const& selection) const
 
 void Layer::erase_selection(Selection const& selection)
 {
-    Gfx::Painter painter { content_bitmap() };
     auto const image_and_selection_intersection = m_image.rect().intersected(selection.bounding_rect());
     auto const translated_to_layer_space = image_and_selection_intersection.translated(-location());
-    painter.clear_rect(translated_to_layer_space, Color::Transparent);
+
+    for (int y = translated_to_layer_space.top(); y < translated_to_layer_space.top() + translated_to_layer_space.height(); ++y) {
+        for (int x = translated_to_layer_space.left(); x < translated_to_layer_space.left() + translated_to_layer_space.width(); ++x) {
+
+            /* Selection is still in pre-translated coordinates, account for this by adding the layer's relative location */
```

Comment:
Please don't use C-style comments

Reply:
Just want to make sure I understand (new to C/C++), it would be acceptable to have that with // instead of /* ... */ correct?

If the comment is multiline is /* */ acceptable or just start each line with //

I can change this here and in my other PixelPaint PRs in a couple hours

Reply:
> it would be acceptable to have that with // instead of /* ... */ correct?

Yes

> If the comment is multiline is /* */ acceptable or just start each line with //

The latter :)

The only time you need `/* */` is in multi-line macros which don't support `//` due to each line having to end with a backslash; and we do it for the license headers as a convention.

Reply:
Thanks for the explanation. I'll get these changed in a bit.

Reply:
I've updated the comment to match the preferred style.